src/chart/bootstrap.ts

```diff
@@ -158,17 +156,6 @@ export default class ChartBootstrap implements ChartContainer {
 			this.canvasModels,
 		);
 		this.mainCanvasModel = mainCanvasModel;
-		const overSeriesCanvasModel = createCanvasModel(
```

Comment:
pls, fix the examples, if there were some with this canvasModel

Reply:
Do we have such examples in dxcharts-lite?

Reply:
I don't know, just asked you to check if it is

Reply:
it's okay 👍  Search through files doesn't show anything